src/ansys/aedt/core/visualization/advanced/misc.py

```diff
@@ -557,32 +557,27 @@ def _write_stl(nas_to_dict, decimation, working_directory, enable_planar_merge=T
     def _write_solid_stl(triangle, pp):
         try:
             # points = [nas_to_dict["Points"][id] for id in triangle]
-            points = [pp[i] for i in triangle]
+            points = [pp[i] for i in triangle[::-1]]
         except KeyError:  # pragma: no cover
             return
-        fc = GeometryOperators.get_polygon_centroid(points)
-        v1 = points[0]
-        v2 = points[1]
-        cv1 = GeometryOperators.v_points(fc, v1)
-        cv2 = GeometryOperators.v_points(fc, v2)
-        if cv2[0] == cv1[0] == 0.0 and cv2[1] == cv1[1] == 0.0:
-            n = [0, 0, 1]  # pragma: no cover
-        elif cv2[0] == cv1[0] == 0.0 and cv2[2] == cv1[2] == 0.0:
-            n = [0, 1, 0]  # pragma: no cover
-        elif cv2[1] == cv1[1] == 0.0 and cv2[2] == cv1[2] == 0.0:
-            n = [1, 0, 0]  # pragma: no cover
-        else:
+        fc = points[0]
+        v1 = points[1]
+        v2 = points[2]
+        cv1 = GeometryOperators.v_points(fc, v2)
+        cv2 = GeometryOperators.v_points(fc, v1)
```

Comment:
```suggestion
        cv1 = GeometryOperators.v_points(fc, v1)
        cv2 = GeometryOperators.v_points(fc, v2)
```
Any reason to changes the name values ? I expected cv1 to be the distance between v1 and the center (same for cv2)

Reply:
it's the sign of the vector that will be impacted